assets/preload/data/controls.txt

```diff
@@ -1,17 +1,17 @@
 setUP
-W
+F
```

Comment:
you mixed up up and down if you are doing `df jk`, then it should have `j` for `up`, and `f` for `down`